docs/.gitattributes

```diff
@@ -0,0 +1,17 @@
+# Auto detect text files and perform LF normalization
+* text=auto
```

Comment:
As specified in the editorconfig line-endings are always lf for all text files

Reply:
updated gitattributes